geomstats/datasets/prepare_emg_data.py

```diff
@@ -0,0 +1,116 @@
+"""Pre-process emg time series into batched covariance matrices.
```

Comment:
NIT: Maybe just give the acronym of EMG, since most users might not be familiar with it?